test/development/acceptance/ReactRefreshLogBox.test.ts

```diff
@@ -462,7 +471,7 @@ describe('ReactRefreshLogBox', () => {
       `
     )
 
-    // TODO(veil): ignore-list Webpack runtime (https://linear.app/vercel/issue/NDX-945)
+    // TODO(veil): ignore-list Turbopack runtime
```

Comment:
Comment is incorrect. Will fix before merge. Turbopack runtime is ignore-list. It just inserts a lot more frames into userspace code.

Reply:
Waiting for https://github.com/vercel/next.js/pull/81300 to land. The runtime was ignore-listed but the magic comment was missing. This wasn't a problem for dev where we fallback to asking the bundler for sourcemaps. But prod relies on the VM knowing where the source map is.